operator/src/test/java/io/strimzi/kafka/access/KafkaAccessReconcilerTest.java

```diff
@@ -71,7 +71,8 @@ public class KafkaAccessReconcilerTest {
 
     @BeforeEach
     void beforeEach() {
-        operator = new Operator(overrider -> overrider.withKubernetesClient(client));
+        operator = new Operator(overrider -> overrider.withKubernetesClient(client)
+                .withUseSSAToPatchPrimaryResource(false));
```

Comment:
IIRC you had comment why the SSA is disabled here, could you add it back?

Reply:
Added a comment